hass/configurations.js

```diff
@@ -3,134 +3,14 @@
 
 module.exports = {
   // Binary sensor https://www.home-assistant.io/components/binary_sensor.mqtt
-  binary_sensor_motion: {
+  binary_sensor: {
     type: 'binary_sensor',
-    object_id: 'motion',
+    object_id: 'event',
     discovery_payload: {
       payload_on: true,
       payload_off: false,
       value_template: '{{ value_json.value }}',
-      device_class: 'motion'
```

Comment:
I would not touch thsi file, ti breaks completely the binary sensors logic. which is using the binary_sensor_<type>

Reply:
We aren't ever using the other ones though. You only ever load binary_sensor_contact above the if block.

Reply:
In this line you can see the second use of this file.
https://github.com/zwave-js/zwavejs2mqtt/blob/3d77e853a826bf990fd6b6861ae04e1d535271a1/lib/Gateway.js#L1602

This is not going to break all binary sensors.

Adding the support for the None change. 
you could probably create a sensor `binary_sensor_none` and use that for notifications to keep the Later `None` removal

Reply:
Ah, ok. I'll stet that back tomorrow.

Reply:
I like this change, it cleans up some repeated values, I would instead fix the other to use the correct device class

Reply:
I got some idea how we can do it! I will send a PR to @blhoward2 's branch for review/merging!